azavea/temperate
# Changes to User model

## Overview

 - Removes the username field from the User model in favor of email address as the identifying/login field
 - Makes first & last name required fields
 - Adds city & organization fields to the username model

## Testing Instructions

 * Run migrations
 * You should be able to create a user via `./scripts/manage createsuperuser` or the Django Admin
 * You should be able to edit users via the Django Admin
 * Attempting to create/update a user with blank first or last name should cause an error

Closes #47 
Closes #48
